.github/workflows/test.yml

```
@@ -2,6 +2,10 @@ name: Code analysis, test and generate sample project
 on:
   pull_request:
     types: [ opened, synchronize, reopened, ready_for_review ]
+    branches-ignore:
+      - main
+      - develop
+      - 'release/*'
```

I expect the test workflow to run on this feature branch `feature/6-set-up-cd-android`. It's not running now https://github.com/nimblehq/kmm-templates/pull/48/commits 🤔 

```suggestion
    push:
      branches-ignore:
        - main
        - develop
        - 'release/**'
```

---

@luongvo That's what I am concerned. 😂

---

Yep, it was not seeing it in detail.